Modules/_ctypes/cfield.c

```diff
@@ -1294,7 +1294,8 @@ s_set(void *ptr, PyObject *value, Py_ssize_t length)
     }
 
     data = PyBytes_AS_STRING(value);
-    size = strlen(data); /* XXX Why not Py_SIZE(value)? */
+    size = strlen(data);
```

Comment:
// bpo-39593: XXX Why not Py_SIZE(value)?

Will be enough information for readers?

Reply:
Thanks, make sense. I update this info later.

Reply:
I think option 1 is enough ;)